monai/bundle/scripts.py

```diff
@@ -547,17 +547,36 @@ def run(
                     },
                     "configs": {
                         "tracking_uri": "<path>",
+                        "experiment_name": "monai_experiment",
+                        "run_name": None,
+                        "is_not_rank0": (
```

Comment:
Hi @wyli , I think we should simplify this example config here to make the doc-string clear, and we will also have a very detailed tutorial link soon.
My proposal example:
```py
tracking = {
    "handlers_id": {"evaluator": {"id": "evaluator", "handlers": "handlers"}},
    "configs": {
        "tracking_uri": "<path>",
        "experiment_name": "monai_experiment",
        "run_name": "my_test_run",
        "evaluator": {
            "_target_": "MLFlowHandler",
            "tracking_uri": "@tracking_uri",
            "experiment_name": "@experiment_name",
            "run_name": "@run_name",
            "iteration_log": False,
        },
    },
}
```
I am not very sure, what do you think from users' point view?

Thanks.

Reply:
(it seems all the bundles have the same set of logging config:
https://github.com/Project-MONAI/model-zoo/blob/dev/models/brats_mri_segmentation/configs/logging.conf maybe we should consider improving the default file as well...)

Reply:
okok, then I am OK with the change.

Thanks.